src/Symfony/Component/Form/Tests/ChoiceList/Factory/CachingFactoryDecoratorTest.php

```diff
@@ -155,127 +155,6 @@ public function testCreateFromChoicesDifferentValueClosure()
         $this->assertSame($list2, $this->factory->createListFromChoices($choices, $closure2));
     }
 
-    public function testCreateFromFlippedChoicesEmpty()
```

Comment:
createFromFlippedChoices is deprecated but doesn't trigger any deprecation (fixed in an other PR)